sourcemash/frontend/static/js/views/sidenav_view.js

```diff
@@ -40,12 +42,21 @@ Sourcemash.Views.SidenavView = Backbone.View.extend({
     }
   },
 
+  showRegisterModal: function(){
+    $("#register-modal").openModal();
+  },
+
+  showForgotPasswordModal: function(){
+    $("#forgot-modal").openModal();
+  },
+
   render: function() {
     this.close();
 
     activeTab = $(".tab .active").text().toLowerCase() || "categories";
 
-    var content = this.template({active: activeTab, current_user: this.user, feeds: this.feeds, categories: this.categories});
+    var content = this.template({active: activeTab, current_user: this.user, feeds: this.feeds,
+                                 categories: this.categories, isLoading: this.loading});
```

Comment:
Padding messed it up because that's how the preloader keeps itself centered. Margin won't cause issues because that pushes surrounding divs away, leaving the space we want. I just tried it locally and it worked.